documentation/book/proc-configuring-kafka-mirror-maker-abort-on-send-failure.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-abort-on-send-failure.adoc
+
+[id='proc-configuring-kafka-mirror-maker-abort-on-send-failure-{context}']
+= Configuring the Mirror Maker to exit on a failed send
```

Comment:
**Configuring whether Mirror Maker exits if messages fail to send**

Reply:
Suggestion 2:

Disable Mirror Maker exit if a message fails to send

Reply:
Suggestion 3:

**Enabling Mirror Maker exit on message failure**